admin/app/controllers/workarea/admin/content_page_copies_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Workarea
+  module Admin
+    class ContentPageCopiesController < Admin::ApplicationController
+      required_permissions :store
+
+      before_action :find_source_page
+
+      def new
+      end
+
+      def create
+        @page_copy =
+          CopyPage.new(@page, params[:page].to_unsafe_h).perform
```

Comment:
Was the `to_unsafe_h` needed here?

Reply:
I used `CatalogProductCopiesController` as my base for this. Copypasta.